assets/xml/objects/object_zl4.xml

```diff
@@ -1,125 +1,177 @@
+        <DList Name="gDmZl4LeftBootDl" Offset="0xD778" />
+
+        <!-- Ocarina in this actor too... couldn't have just made it a different object? why do we need like 4 copies of this.. -->
+        <Texture Name="gDmZl4OcarinaTex" OutName="ocarina" Format="rgba16" Width="32" Height="16" Offset="0xD8B8" />
+        <DList Name="gDmZl4OcarinaDl" Offset="0xDE08" />
+
+        <!-- Child Zelda Skeleton, Copy of OoT -->
+        <Limb Name="gDmZl4RootLimb" Type="Standard" EnumName="ZL4_LIMB_ROOT" Offset="0xDF28" />
+        <Limb Name="gDmZl4WaistLimb" Type="Standard" EnumName="ZL4_LIMB_WAIST" Offset="0xDF34" />
+        <Limb Name="gDmZl4LeftLegLimb" Type="Standard" EnumName="ZL4_LIMB_LEFT_LEG" Offset="0xDF40" />
+        <Limb Name="gDmZl4LeftFootLimb" Type="Standard" EnumName="ZL4_LIMB_LEFT_FOOT" Offset="0xDF4C" />
+        <Limb Name="gDmZl4RightLegLimb" Type="Standard" EnumName="ZL4_LIMB_RIGHT_LEG" Offset="0xDF58" />
+        <Limb Name="gDmZl4RightFootLimb" Type="Standard" EnumName="ZL4_LIMB_RIGHT_FOOT" Offset="0xDF64" />
+        <Limb Name="gDmZl4Dress1Limb" Type="Standard" EnumName="ZL4_LIMB_DRESS_1" Offset="0xDF70" />
+        <Limb Name="gDmZl4Dress2Limb" Type="Standard" EnumName="ZL4_LIMB_DRESS_2" Offset="0xDF7C" />
+        <Limb Name="gDmZl4Dress3Limb" Type="Standard" EnumName="ZL4_LIMB_DRESS_3" Offset="0xDF88" />
+        <Limb Name="gDmZl4TorsoLimb" Type="Standard" EnumName="ZL4_LIMB_TORSO" Offset="0xDF94" />
+        <Limb Name="gDmZl4LeftUpperArmLimb" Type="Standard" EnumName="ZL4_LIMB_LEFT_UPPER_ARM" Offset="0xDFA0" />
+        <Limb Name="gDmZl4LeftForarmLimb" Type="Standard" EnumName="ZL4_LIMB_LEFT_FORARM" Offset="0xDFAC" />
+        <Limb Name="gDmZl4LeftHandLimb" Type="Standard" EnumName="ZL4_LIMB_LEFT_HAND" Offset="0xDFB8" />
+        <Limb Name="gDmZl4RightUpperArmLimb" Type="Standard" EnumName="ZL4_LIMB_RIGHT_UPPER_ARM" Offset="0xDFC4" />
+        <Limb Name="gDmZl4RightForarmLimb" Type="Standard" EnumName="ZL4_LIMB_RIGHT_FORARM" Offset="0xDFD0" />
+        <Limb Name="gDmZl4RightHandLimb" Type="Standard" EnumName="ZL4_LIMB_RIGHT_HAND" Offset="0xDFDC" />
+        <Limb Name="gDmZl4HeadLimb" Type="Standard" EnumName="ZL4_LIMB_HEAD" Offset="0xDFE8" />
+        <Skeleton Name="gZl4Skeleton" Type="Flex" LimbType="Standard" LimbNone="ZL4_LIMB_NONE" LimbMax="ZL4_LIMB_MAX" EnumName="zl4_Limbs" Offset="0xE038" />
+
+        <!-- unused, left over from OoT I think  -->
+        <Animation Name="gDmZl4TurningAroundAnim" Offset="0xF250" />
+        <Animation Name="gDmZl4IdleHandsInFrontAnim" Offset="0xFDC8" />
+        <Animation Name="gDmZl4RaiseFluteToPlayAnim" Offset="0x10334" />
+        <Animation Name="gDmZl4LowerFluteAfterPlayAnim" Offset="0x105C8" />
+        <Animation Name="gDmZl4PlayingFluteAnim" Offset="0x10D50" />
+        <Animation Name="gDmZl4FacingAwayIdle" Offset="0x11294" />
+
+        <!-- Used in MM -->
+        <Animation Name="gDmZl4GivingItemToLinkStartAnim" Offset="0x11BB8" />
```

Comment:
Just thinking about shortening this a bit; does the name need to reference Link? Couldn't it just be `gDmZl4GivingItemStartAnim`? If she was giving an item to a different character, would she use a different animation?

Reply:
makes sense, will do